server/insertPost.js

```diff
@@ -1,133 +1,137 @@
 #!/usr/bin/env node
 
-"use strict";
+"use strict"
 
-var fs       = require("fs")
-var path     = require("path")
-var config   = require("../config")
-var shortid  = require("shortid")
-var utils    = require("./utils")
-var logger   = require("./logger")
+const fs       = require("fs")
+const path     = require("path")
+const config   = require("../config")
+// const shortid  = require("shortid")
+// const utils    = require("./utils")
+const logger   = require("./logger")
 
-var prompt
-var globals  = {}
+let prompt
```

Comment:
I'd prefer an assignment here (`let prompt = undefined`) even if it's functionally identical.